src/isomorphic/React.js

```diff
@@ -109,8 +110,11 @@ if (__DEV__) {
       get() {
         lowPriorityWarning(
           didWarnPropTypesDeprecated,
-          'Accessing PropTypes via the main React package is deprecated. Use ' +
-            'the prop-types package from npm instead.',
+          'In React v16.0+, accessing PropTypes via the main React package is deprecated. Use ' +
```

Comment:
Isn't this implying it *got deprecated* in 16 but will be removed later? Usually we say something is deprecated in the current cycle and *removed* (or moved) in the next cycle. Otherwise it seems like we have not deprecated it yet (but we did).